service.subtitles.supersubtitles/changelog.txt

```diff
@@ -0,0 +1,93 @@
+0.0.23
```

Comment:
The changelog is no longer parsed by kodi. You can still keep it in the submission as a global changelog, but Kodi now has a field in addon.xml dedicated to this. You can add:

```xml
<news>
    0.0.23:
      - archive handling changes
      - leia addon related changes
      - inital python3 support
</news>
```
To the extension.metadata of your addon.xml. This news field is dedicated to the changelog of the last version only.

Reply:
I know that.

Reply:
The changelog.txt is no longer used by Kodi and should not be included in an add-on. If you want user to see it, has to be put in the addon.xml `<news>`. If you don't think a users need this, then don't submit the changelog.txt.